src/context-items/rename.ts

```diff
@@ -0,0 +1,30 @@
+import * as _ from 'lodash';
+import { Meta } from '../base';
+import { Context } from '../context';
+import { ContextItem } from './context-item';
+
+export class Rename implements ContextItem {
+  constructor(readonly newPath: string) {}
+
+  async run(context: Context, value: any, meta: Meta) {
+    const { req, location, path } = meta;
+
+    if (path !== this.newPath) {
+      if (context.fields.length !== 1) {
+        throw new Error('Cannot rename multiple fields.');
+      }
+
+      if (this.newPath.includes('*')) {
+        throw new Error('Cannot use rename() with wildcards.');
+      }
```

Comment:
WDYT of moving this to the constructor/`.rename()`?

Just thinking that failing on server start might be a better developer experience than on request.